presto-test-jdbc-compatibility-old-server/testng.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<!--
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<suite name="JdbcCompatibilytOldServer" verbose="1" >
+    <test name="all" group-by-instances="true">
```

Comment:
I remember you told me it  is needed but

- i dont remember why, this needs to be doc'd
- i still would try to remove it (i can try if you want me to)

Reply:
Without `group-by-instances="true"` tests are grouped by test method. Like that:
```
instance_a.testA()
instance_b.testA()
instance_c.testA()
instance_a.testB()
instance_b.testB()
instancd_c.testB()
```
This is very resource intensive as each initialized instance is separate running docker instance.

With this flag only one docker instance is running as exutions are grouped by instance.

I will see if hack to enforce single threaded execution may solves this one too.

Reply:
> This is very resource intensive as each initialized instance is separate running docker instance.

Alas! ... please document that